app/models/repository_host/github.rb

```diff
@@ -62,7 +62,7 @@ def self.fetch_repo(id_or_name, token = nil)
       hash[:keywords] = hash[:topics]
       hash[:host_type] = "GitHub"
       hash[:scm] = "git"
-      hash[:status] = "Unmaintained" if hash[:archived]
+      hash[:status] = hash[:archived] ? "Unmaintained" : nil
```

Comment:
One concern I have is this status could potentially overwrite other statuses outside of unmaintained. Another option I had in the works was to make a specific `archived` field and have `RepositoryHost` set the unmaintained status specifically based on the value of that field instead of doing what I have here now.

Reply:
I think I'm going to go ahead and make this change so that this code doesn't overwrite existing statuses.